src/test/mod.rs

```diff
@@ -202,14 +204,20 @@ fn init_env(project_dir: &str) {
     env::set_current_dir(cwd).expect(FAIL_MSG);
 }
 
+#[derive(Clone, Debug, PartialEq)]
+enum Id {
```

Comment:
I don't think you need either. I think that each test case should be able to specify an id in ad hoc manner. I.e., they would only test the id if that is what they are testing, if it doesn't matter, then they can use id 0 or 42 or whatever